modules/microshift-blocking-nodeport-access.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// * microshift_networking/microshift-networking.adoc
+
+:_content-type: PROCEDURE
+[id="microshift-blocking-nodeport-access_{context}"]
+= Blocking external access to the NodePort service on a specific host interface
+
+OVN-Kubernetes does not restrict the host interface, therefore NodePort services can be accessed from outside a {product-title} node. The following procedure explains how to block the NodePort service on a specific host interface and restrict external access.
```

Comment:
```suggestion
OVN-Kubernetes does not restrict the host interface where a NodePort service can be accessed from outside a {product-title} node. The following procedure explains how to block the NodePort service on a specific host interface and restrict external access.
```

Reply:
NodePort service is designed to be accessed from outside the cluster node, the `therefore` in above sentence may not be accurate in my opinion.  For security reason, we add this feature to allow user to restrict which host interface the nodeport service can be accessed. hope it makes sense.

Reply:
Ohh I gotcha! I was misunderstanding the content, thank you!